tr069/src/main/java/com/github/freeacs/tr069/Properties.java

```diff
@@ -157,4 +162,6 @@ public int getConcurrentDownloadLimit() {
   public String getPublicUrl() {
     return publicUrl;
   }
+
+  public boolean wantsAppendHwVersion() { return appendHwVersion; }
```

Comment:
not too sure about the wants prefix. it should be either getAppendHwVersion, isAppendHwVersion or shouldAppendHwVersion. I think either the first one or the second one is the better choices. We are in code land here, and and code must be consistent :) the second one would read "if properties is append hw version". your version is "if properties wants hw version".  So i think we should go for is or get. is preferably.

Reply:
Ok, my first thought was isAppendHwVersion, but shouldAppendHwVersion sounds better :)